CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# librdkafka v2.5.1
```

Comment:
Change patch version because of the .NET 2.5.1 and 2.5.2 releases
```suggestion
# librdkafka v2.5.3
```

Reply:
Already did in the airlock PR. Please check.